### PR TITLE
Compeition branch

### PR DESCRIPTION
**Pull request by JasonGibson274**
Open questions: why the gps conversion was removed

Otherwise this is mostly fixing or removing bad code that I wrote at competition/before that should never have been merged into master. Also updating the urdf for Woodi sensor position wise. If we really want to update the urdf to mimic the actual robot then add an issue and assign it to me but I would rather walk someone else (freshmen) through setting up the new robot.

### CODE REVIEW

**Comment by JasonGibson274** [comment]
It appears that the UTM conversion node does not support kinetic and the only method we needed was copied into a file. Therefore we do not need this.

**Comment by dpattison3** [comment]
We also have the nav sat transform node that launches with the localization node that performs a similar thing I think.

**Comment by JasonGibson274** [comment]
The header file is only used to convert the latitude and longitude of the waypoint into UTM.

**Comment by JasonGibson274** [comment]
Agreed I will double check the sensors are being launched in a logical fashion and work as intended and I will remove the waypoints file and make a single one.

**Comment by JasonGibson274** [comment]
@rmkeezer we did not have the side cameras attached and we are not really planning on it for the new robot. If we add them in we can revert back to the default being all three.

File: sandbox/qual_waypoints.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+42.67833241, -83.19551079
```
Comment (JasonGibson274) [inline sandbox/qual_waypoints.csv]: remove excess waypoints files.

File: sandbox/test_waypoints.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+42.67858287, -83.19530227
```
Comment (JasonGibson274) [inline sandbox/test_waypoints.csv]: remove excess waypoints files.

File: igvc/launch/sensors.launch [hunk 3]
```diff
@@ -6,22 +6,25 @@
 <launch>
```
Comment (JasonGibson274) [inline igvc/launch/sensors.launch]: Check that launch file work as intended.

**Pull request merged**